### ISSUE TITLE
UploadOnSave not working

### ISSUE THREAD

**Issue by dbkreling** [dingjie/atom-sync#54]
UploadOnSave does not work for me despite having `uploadOnSave: true`on my '.sync-config.cson'. It first stopped working on an active project, then it stopped working on newly created projects as well.

Manually syncing does work.

I quickly tried to map a keybind for 'upload-file' without success, and did not find much information online.

Is this an issue (for which I can provide more info) or should I try a workaround (such as a keybind)?
Please let me know what info you need. Thank you.

**Comment by pmgouveia** [comment]
yeah happen to me too... any further solution? How is it possible to debug this plugins?

**Comment by pmgouveia** [comment]
weird.. restart many times.. and nothing.. now updated git-control and works again
